### PR TITLE
Internal server error no default value

### PR DESCRIPTION
**Pull request by samhstn**
Need to rebase after #276 

See #267

### CODE REVIEW

**Review by Danwhy — APPROVED** [review]

**Comment by SimonLab** [comment]
@shouston3 #276 is merged, do you want to rebase?

**Comment by shouston3** [comment]
@SimonLab rebased

File: web/controllers/spreadsheet_controller.ex [hunk 1]
```diff
@@ -28,11 +28,10 @@ defmodule App.SpreadsheetController do
     do: handle_submit conn, :suggestions, [suggestions], path
   def submit(conn, %{"email" => %{"email" => email}}, path),
     do: handle_submit conn, :emails, [email], path
-  def submit(conn, %{"feedback" => %{"email" => email,
-                                     "question1" => question1,
-                                     "feedback1" => feedback1,
-                                     "feedback2" => feedback2}}, path) do
-    input_list = [question1, feedback1, feedback2, email]
+  def submit(conn, %{"feedback" => feedback}, path) do
+    input_list = Enum.map ["email", "feedback1", "feedback2"], &(feedback[&1])
```
Comment (SimonLab) [inline web/controllers/spreadsheet_controller.ex]: Are email, feedback1 and feedback2 always defined or do we want to have some default values for these input too? If they are not defined ```feedback[&1]``` will return some ```nul``` values

**Review by SimonLab — COMMENTED** [review]
LGTM, just one comment/question to clarify the list of valid inputs

File: web/controllers/spreadsheet_controller.ex (hunk 1 above)
Comment (shouston3) [inline web/controllers/spreadsheet_controller.ex]: They will always come through. If they are left blank when submitting the form, then they will be an empty string

**Review by SimonLab — APPROVED** [review]

**Pull request merged**